Makefile

```diff
@@ -13,7 +13,7 @@ WERROR ?= 0
 # Keep .mdebug section in build
 KEEP_MDEBUG ?= 0
 # Disassembles all asm from the ROM instead of skipping files which are entirely in C
-FULL_DISASM ?= 0
+FULL_DISASM ?= 1
```

Comment:
Not really opposed myself, but should probably have a full discussion on Discord about disassembling everything being the default. In particular, I don't think Jenkins should fully disassemble each time so the Jenkinsfile should be adjusted to pass `FULL_DISASM=0`.

Reply:
Oops, left that in by accident.